rust-toolchain

```diff
@@ -0,0 +1 @@
+1.72.0
```

Comment:
Maybe update CI [RUST_TOOLCHAIN: 1.68.2](https://github.com/synapseweb3/crosschain-ckb-contracts/blob/e8e4a12bab6fb7e665f6b2556896f181ffd0b1c0/.github/workflows/ci.yaml#L14) too?

Reply:
I'd also need to update rust in the docker image.